frontend/public/components/utils/kebab.tsx

```diff
@@ -61,32 +62,28 @@ export const KebabItems: React.SFC<KebabItemsProps> = ({options, onClick, isActi
   </ul>;
 };
 
+export const buildAccessReview = (kind: K8sKind, obj: K8sResourceKind, verb: K8sVerb): AccessReviewResourceAttributes => ({
```

Comment:
This might be better in `frontend/public/components/utils/rbac.tsx` since it's not specific to kebabs.

I would call `kind` either `model` or `kindObj` to avoid confusion with a kind string. (Upstream uses `kind` to refer to strings like `'Pod'`.) I know this is what we use `kind` below in the kebab functions below, but we should try to be more precise in new code.

`buildAccessReview` sounds like a review for a `Build` resource. Maybe something like `toAccessReview` / `asAccessReview` / `makeAccessReview` ?

Reply:
Thanks, moved and renamed